cg/apps/janus/api.py

```diff
@@ -0,0 +1,28 @@
+"""The JanusAPIClient."""
```

Comment:
Consider creating a new directory called clients and placing this one in there. This is a new specific thing, I think we should separate it from the existing apps (even though some of the existing apps basically are web api clients).